opentreemap/importer/errors.py

```diff
@@ -83,7 +83,9 @@ def is_itree_error_code(code):
 MERGE_REQUIRED = e(71, trans('This row must be merged'), False)
 
 NEARBY_TREES = e(
-    1050, trans('There are already trees very close to this one'), False)
+    1050, trans('There are already trees within ten feet of this one'), False)
```

Comment:
I'm a little leery of making this change.

It may be a little jarring for maps using metric units to see imperial units in their error messages when we require metric units in their upload.